src/app/api/new-quiz/route.ts

```diff
@@ -28,12 +28,12 @@ export async function POST(req: NextRequest) {
 
   try {
     // Get topic details first
-    const topic = await getTopicById(parseInt(topicId));
+    const topic = await getTopicById(topicId);
```

Comment:
why change this? the getTopicById still expects a number

Reply:
Yeah, right, this was a mistake